### ISSUE TITLE
[A11yMAS][Accessibility][Keyboard][MAS3.2.2][Eclipse][Wipro]: ToolTip (Information Icon)is not accessible which is present in "Work With " Edit field in "Available Software Window"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1348]
**Repro Steps:**

1. Open and Start Eclipse.
2. Press "Alt+H" to open Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter
5. Then Navigate to "Work With" edit field then try to access the information icon(i) then observe the keyboard behavior

**Expected Result:**
ToolTip Information Should be accessible by using keyboard

**Actual Result:**
ToolTip information is not accessible by using keyboard in available software window

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(OS Build 16299.192)

![tooltip is not accessible by using keyboard in available software window](https://user-images.githubusercontent.com/22956130/35389088-591abb3e-01fc-11e8-81c2-52f9203809c0.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=534670

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's about the one-time installation.
* **Quality of experience**: not impacted. A simple workaround is available, users can install the plugin from **Help->Marketplace** instead of **Help->Install New Software**.  
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749831
